test/scripts/agent-images/create_agent_images.sh

```
@@ -27,35 +28,56 @@ if [ -n "${FLIGHTCTL_RPM:-}" ]; then
-        fi
-
-        podman build ${args:+${args}} -f "${containerfile_path}" -t "${container_name}" .
-        podman tag "${container_name}" "${FINAL_REF}"
-        podman push "${FINAL_REF}"
-    done
+    # Build base image first (required by others)
+    echo -e "\033[33mBuilding base image first...\033[m"
+    build_single_image "base"
+
+    # Build remaining images in parallel
+    local other_images=$(echo $IMAGE_LIST | sed 's/base//')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)